docs/mine/lotus/split-markets-miners.md

```diff
@@ -68,51 +104,66 @@ export LOTUS_BACKUP_BASE_PATH=/tmp
+If you intend to run the `mining/sealing/proving` node on the same machine as the
+`markets` node, make sure that their listeners do not clash.
+By default the `lotus-miner` API server listens to port 2345, so in the example
+configuration above, we change the configuration for the `markets` node API server
+to listen to `127.0.0.1:8787`.
 
-Make sure you adjust the `[Libp2p]` section on the `markets` node accordingly - it needs to be publicly accessible, so that clients can make storage and retrieval deals with your system.
+Make sure you adjust the `[Libp2p]` section on the `markets` node accordingly - it
+needs to be publicly accessible so that clients can make storage and retrieval
+deals with your system.
 
-The `[Libp2p]` section on the `mining/sealing/proving` node can be removed because this node will no longer be running a Libp2p node.
+The `[Libp2p]` section of the `config.toml` on the `mining/sealing/proving` node
+can be removed because it will no longer be running a Libp2p node.
 
-### 3. Initialising a `markets` service repository
+#### 3. Initialising a `markets` service repository
 
-1. Create authentication tokens for the `markets` node. This is an online operation, so your `lotus-miner` should be running with its updated configuration (in case you move its API to another IP:PORT)
+1. Create authentication tokens for the `markets` node. This is an online operation,
+so your `lotus-miner` should be running with its updated configuration (in case
+you move its API to a different IP:PORT)
 
 ```shell
 export APISEALER=`lotus-miner auth api-info --perm=admin`
 export APISECTORINDEX=`lotus-miner auth api-info --perm=admin`
 ```
 
-2. Initialise the `market` node. This performs a one-time setup of the markets node. Part of that setup includes updating the `peer id` in the miner actor by submitting a message on chain. This is necessary so that storage and retrieval clients know that this miner's **deal-making** endpoint is now publicly dialable/reachable on a new address (the new `market` node).
-
-Note that `lotus-miner` interacts with one repository or another depending on the `LOTUS_MINER_PATH` environment variable!
+2. Initialise the `market` node. This performs a one-time setup of the markets node.
+Part of that setup includes updating the `peer id` in the miner actor by submitting
+a message on chain. This is necessary so that storage and retrieval clients know that
+this miner's **deal-making** endpoint is now publicly dialable/reachable on a new
+address (the new `market` node).
 
-This command should be run on the `markets` miner instance host, as it is creating the `markets` miner instance repository, among other actions.
+This command should be run on the `markets` miner instance host, as it is creating
+the `markets` miner instance repository, among other actions.
 
 ```shell
 lotus-miner --markets-repo=~/.lotusmarkets init service --type=markets \
```

Comment:
We should probably change this command to `lotus-miner init subsystem`, if we're settling on subsystems as the canonical name for these. The term `service` hasn't appeared anywhere up until now...